core/options.js

```diff
@@ -171,7 +173,7 @@ Blockly.Options = function(options) {
   /**
    * The parent of the current workspace, or null if there is no parent
    * workspace.
-   * @type {Blockly.Workspace}
+   * @type {Blockly.WorkspaceSvg}
```

Comment:
Maybe add a comment explaining why we can assume that this is a WorkspaceSvg.

Reply:
Done. Please check if that makes sense.